gateway/src/config/template.rs

```diff
@@ -73,6 +77,9 @@ keys.public_sphinx_key_file = '{{ storage_paths.keys.public_sphinx_key_file }}'
 # derived shared keys and available client bandwidths.
 clients_storage = '{{ storage_paths.clients_storage }}'
 
+# Path to the configuration of the locally running network requester.
```

Comment:
imho we should probably call it embedded or something, locally to me just means running on the same machine